base/markdown/Common/inline.jl

```diff
@@ -106,6 +106,42 @@ function footnote_link(stream::IO, md::MD)
     end
 end
 
+@trigger '<' ->
+function autolink(stream::IO, md::MD)
+    withstream(stream) do
+        startswith(stream, '<') || return
+        url = readuntil(stream, '>')
```

Comment:
Are we hitting this definition already? https://github.com/JuliaLang/julia/blob/550f5d33cad1396e6e9a84843bddc8cc9b4702ed/base/markdown/parse/util.jl#L139-L159

I guess I haven't fully grokked what match/count do there, but it looked to me like it was suggesting you'd use `<` as the match character i.e. matching braces.